azure-sfmesh-maven-plugin/src/main/java/com/microsoft/azure/maven/servicefabric/Utils.java

```diff
@@ -0,0 +1,216 @@
+            throw new MojoFailureException(String.format("Error while " +
+                "deleting directory %s", path));
+        }
+    }
+
+    public static String replaceString(Log logger, String content, String originalString,
+        String replaceString, String resourceFileName){
+        logger.debug(String.format("replacing %s with %s in %s",
+            originalString, replaceString, resourceFileName));
+        return content.replace(originalString, replaceString);
+    }
+
+    public static String getServicefabricResourceDirectory(Log logger,
+        MavenProject project) throws MojoFailureException {
+        return Paths.get(project.getBasedir().toString(), "servicefabric").toString();
+    }
+
+    public static String getAppResourcesDirectory(Log logger,
+        MavenProject project) throws MojoFailureException {
+        return Paths.get(getServicefabricResourceDirectory(logger, project),
+            "appresources").toString();
+    }
+
+    public static String getPath(String directoryPath, String fileOrDirName){
+        return Paths.get(directoryPath, fileOrDirName).toString();
+    }
+
+    public static String executeCommand(Log logger, String command) throws MojoFailureException{
+        try {
+            logger.info(String.format("Executing command %s", command));
+            final Process p;
+            if (Utils.isWindows()){
+                p = Runtime.getRuntime().exec("cmd.exe /C" + command);
+            } else {
+                p = Runtime.getRuntime().exec(command);
+            }
+            p.waitFor();
+            final int exitCode = p.exitValue();
+            final String stderr = IOUtil.toString(p.getErrorStream(), "UTF-8");
+            final String stdout = IOUtil.toString(p.getInputStream(), "UTF-8");
+            logger.debug(String.format("STDOUT: %s", stdout));
+            if (stderr != null && stderr.length() > 0){
+                if (exitCode != 0){
+                    logger.error(String.format("Process exited with exit code %d", exitCode));
+                    logger.error(String.format("If STDERR: %s", stderr));
+                    throw new MojoFailureException(String.format("Error while " +
+                        "running the %s command", command));
+                } else {
+                    logger.info(String.format("Else STDERR: %s", stderr));
+                }
+            }
+            return stdout;
+        } catch (IOException e){
+            logger.error(e);
+            throw new MojoFailureException(String.format("Error while " +
+                "running the %s command", command));
+        } catch (InterruptedException e) {
+            logger.error(e);
+            throw new MojoFailureException(String.format("Interrupted while " +
+                "running command %s", command));
+        }
+    }
+
+    public static String executeCommand(Log logger, String[] command) throws MojoFailureException{

```

Comment:
Is this method used anywhere?

Reply:
removed this as this is not used anywhere